src/modules/addTask.js

```diff
@@ -24,22 +30,29 @@ const addTask = (task) => {
   taskContent.appendChild(span);
   const li = document.createElement('li');
   li.classList.add('list');
-  li.id = task.index;
+  li.id = `${index}`;
   li.draggable = true;
   li.appendChild(taskContent);
   container.appendChild(li);
 
-  const inputChecked = (task) => {
+  const inputChecked = () => {
     if (input.checked) {
       label.classList.add('checked');
-      task.bool = 'true';
+      tasks[index].bool = true;
     } else {
       label.classList.remove('checked');
-      task.bool = 'false';
+      tasks[index].bool = false;
     }
   };
 
+  const remove = () => {
+    container.removeChild(li);
+    removeTask(li, tasks);
+  };
   input.addEventListener('click', inputChecked);
+  span.addEventListener('click', () => {
+    remove();
```

Comment:
- [ ] As part of the requirement the const remove should take the **clear all completed button** (if the check-box = true => remove item from the array), but **the clear all completed btn** is not working and the **trash icon** it remove everything.

Reply:
No the trash icon don't remove everything. I just implemented the feature of showing a cross line and changing bool value.

Reply:
The Clear all button is intended for the milestone 3 (next milestone), please don't invalidate me for that reason